src/box/engine.h

```diff
@@ -94,6 +94,11 @@ engine_backup_cb(const char *path, void *arg);
 struct engine_vtab {
 	/** Destroy an engine instance. */
 	void (*free)(struct engine *);
+	/**
+	 * Shutdown an engine instance. Shutdown is stopping all internal
```

Comment:
Nit: 'stops' not 'stopping' (here and in other comments). I'd mention that it may yield.